packages/wdio-browserstack-service/src/reporter.ts

```diff
@@ -267,7 +267,7 @@ class _TestReporter extends WDIOReporter {
                 browser: this._capabilities?.browserName,
                 browser_version: this._capabilities?.browserVersion,
                 platform: this._capabilities?.platformName,
-                platform_version: getPlatformVersion(this._userCaps as WebdriverIO.Capabilities)
+                platform_version: getPlatformVersion(this._capabilities)
```

Comment:
Can you quickly check if we have fallbacks in other frameworks? If yes, then we will need to add fallback to keep consistency. Please check Java or Python or C#

Reply:
C# does not seem to have, getting from driver only 
https://github.com/browserstack/browserstack-csharp-sdk/blob/f367e6deb93e8ed9c0af6effe80912776957aece/BrowserStackSDK/Accesibility/Context.cs#L126